billing-report-agent/src/main/java/com/epam/pipeline/billingreportagent/service/impl/converter/StorageToBillingRequestConverter.java

```diff
@@ -138,7 +133,11 @@ public AwsStorageToBillingRequestConverter(final AbstractEntityMapper<StorageBil
     }
 
     private Optional<Long> extractStorageSize(final SearchResponse response) {
-        final ParsedSum sumAggResult = response.getAggregations().get(STORAGE_SIZE_AGG_NAME);
+        final Aggregations aggregations = response.getAggregations();
```

Comment:
This method maybe simplefied:
`Optional.ofNullable(aggregations).map ...

Reply:
Done